front/lib/agent_yaml_converter/converter.ts

```diff
@@ -180,6 +161,23 @@ export class AgentYAMLConverter {
                 : undefined,
               time_frame: action.configuration.timeFrame || undefined,
               json_schema: action.configuration.jsonSchema || undefined,
+              reasoning_model: action.configuration.reasoningModel
```

Comment:
Noob question: do we need to destructure the `action.configuration.reasoningModel`? Meaning couldn't we simply do?
```
reasoning_mode: action.configuration.reasoningModel
```